package.json

```diff
@@ -37,6 +37,9 @@
     "@types/styletron-engine-atomic": "^1.1.1",
     "@types/styletron-react": "^5.0.3",
     "@types/styletron-standard": "^2.0.2",
+    "husky": "^7.0.0",
+    "prettier": "^2.3.2",
+    "pretty-quick": "^3.1.1",
```

Comment:
Would love to understand the reason behind using "pretty-quick" instead of "lint-staged". As create-react-app ships with ESLint, would "lint-staged" be something to consider instead: https://prettier.io/docs/en/precommit.html

Separating linting from formatting maybe?

Reply:
Hmmm, I was ignoring eslint as the original issue did not mention it. Can for sure move it to lint-staged though!

Reply:
Yeah, it seems like eslint hasn't been configured yet as well, issue https://github.com/CovidEngine/vaxxnz/issues/88. But at least prettier precommit is configured so that it can gel nicely with eslint.